polymode-compat.el

```diff
@@ -325,7 +325,8 @@ changes."
           (evil-change-state old-state))))))
 
 (eval-after-load 'evil-core
-  '(add-hook 'polymode-after-switch-buffer-hook 'polymode-switch-buffer-keep-evil-state-maybe))
+  '(add-hook 'polymode-after-switch-buffer-hook 'polymode-switch-buffer-keep-evil-state-maybe)
+  '(add-hook 'polymode-init-inner-hook #'evil-normalize-keymaps))
```

Comment:
This is probably ok, but can this be moved to the `polymode-switch-buffer-keep-evil-state-maybe` somehow? Basically, is there a way to check if keymaps are not installed and call `evil-normalize-keymaps` conditionally only when needed? 

I think this problem occurs only when `old-state` is the same as `new-state` and is 'normal. I am just concerned that such unconditional keymap normalization can mess up with people who have evil loaded but not using it. What do you think?

Reply:
@srnnkls any ieas?